### ISSUE TITLE
Incompatibility with stricter CSPs (Content Security Policies) inline-style

### ISSUE THREAD

**Issue by Jaaap** [sa-si-dev/virtual-select#72]
When you use this lib in a webpage with a strict CSP header, like:
```
Content-Security-Policy: default-src  'self'; script-src 'self'; style-src 'self'; img-src 'self' data:;
```
you get errors like:
```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“style-src”). Source: height: 40px;. virtualselect.js:654:3
```
because this CSP blocks code like:
```javascript
div.innerHTML = '<span style="color:red">Red</span>'
```

Is it feasible to set these styles in JS after the nodes have been made?

**Comment by sa-si-dev** [comment]
@Jaaap I will check and fix it in the future release.

**Comment by sa-si-dev** [comment]
@Jaaap this issue got fixed in the [release v1.0.19](https://github.com/sa-si-dev/virtual-select/releases/tag/v1.0.19)
